app/services/index.ts

```diff
@@ -23,6 +23,7 @@ export * from "./vm-size.service";
 export * from "./adal";
 export * from "./batch-client.service";
 export * from "./storage-client.service";
+export * from "./predefined-formula.service";
 
```

Comment:
tslint checks import sort, but not for export, I guess there are some dependencies?